src/plugins/scopes/ScopesPlugin.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+
+import BasePlugin from '../BasePlugin';
+import ScopesApi from './ScopesApi';
+import type Program from '../../elements/types/Program';
+import type Node from '../../elements/Node';
+
+export default class ScopesPlugin extends BasePlugin {
```

Comment:
For abstraction. `BasePlugin` is abstract class and every plugin defined inside `cst` package should extend it.